## File "./flocq/Core/Raux.v", line 2279, characters 8-15:
## Error: The variable lt_0_Sn was not found in the current environment.

Index: flocq/Core/Raux.v
--- flocq/Core/Raux.v.orig
+++ flocq/Core/Raux.v
@@ -2276,7 +2276,7 @@ assert (Hi: forall n, (0 < INR n + 1)%R).
   intros N.
   rewrite <- S_INR.
   apply lt_0_INR.
-  apply lt_0_Sn.
+  apply Nat.lt_0_succ.
 intros P HP.
 set (E y := exists n, (P n /\ y = / (INR n + 1))%R \/ (~ P n /\ y = 0)%R).
 assert (HE: forall n, P n -> E (/ (INR n + 1))%R).
